packages/components/src/popover/index.js

```diff
@@ -61,12 +61,30 @@ function computeAnchorRect(
 			return;
 		}
 
-		if ( anchorRef instanceof window.Range ) {
+		if ( anchorRef.endContainer ) {
 			return getRectangleFromRange( anchorRef );
 		}
 
-		if ( anchorRef instanceof window.Element ) {
-			const rect = anchorRef.getBoundingClientRect();
+		const { ownerDocument } = anchorRef;
+
+		if ( ownerDocument ) {
+			let rect = anchorRef.getBoundingClientRect();
+
+			if ( ownerDocument !== document ) {
+				const iframe = Array.from(
+					document.querySelectorAll( 'iframe' )
```

Comment:
This would be a bit faster:
```suggestion
					document.getElementsByClassName( 'iframe' )
```

Reply:
You mean `getElementsByTagName`? Yes, probably, I'm not sure if it would be very noticeable though.

Reply:
@ellatrix yeah, that's what I meant. I'll fix the suggestion(s). And from what I've seen it actually does make a speed difference since it doesn't have to do as much parsing from the input string.